oeps/processes/oep-0060-proc-sec-group.rst

```diff
@@ -51,52 +51,96 @@ Reports come into security@openedx.org
-  * Get ticket prioritized and resolved (i.e. project manage the ticket(s) in SME backlogs)
+The security working group gives security severity and remediation advice to maintainers
+----------------------------------------------------------------------------------------
+The security working group is responsible for advising on security and providing security-scoring-as-a-service for maintainers.
 
-    * Maintainers are responsible for resolving security issues
-    * Security Group will set security SLA to manage expectations for time to resolve
-  * Work with SMEs to release the change to master and current supported Open edX release
+Maintainers are encouraged and welcome to email security@openedx.org with any security questions they would like advice on. The security working group will do their best to provide good security advice.
 
-Because the on-call member is expected to follow a disclosure through its eventual resolution, some of these
-responsibilities will extend beyond the initial scheduled on-call time.
+The security working group will also score incoming vulnerabilities using the `Common Vulnerability Scoring System (CVSS), version 3.1`_. This system provides a way to score vulernabilities from 0.0 to 10.0, which map to a severity of Low, Medium, High, or Critical.
+
+This numerical score and qualitative severity rating will be provided to maintainers so they are better informed on how severe the vulnerability is and have a sense of how important it may be to interrupt their work on the repository to resolve the vulnerability.
+
+.. _Common Vulnerability Scoring System (CVSS), version 3.1: https://www.first.org/cvss/v3.1/specification-document
+
+Maintainers are ultimately responsible for the security of their repositories
+-----------------------------------------------------------------------------
+Maintainers are responsible for resolving disclosed security vulnerabilies and notifying the security working group once they are resolved.
+
+To assist the maintainer with keeping track of their repository's vulnerabilities, the security working group will create a `GitHub repository security advisory`_ pre-assigned with the scored severity.
+
+Maintainers are requested to notify the security working group if they would like to adjudicate a vulnerability to a different severity than what was originally triaged. The security working group must accept the maintainer's adjudication, but should comment on any considerations around the adjudication.
+
+Maintainers will be reminded to remediate disclosures in proportion to the severity of the disclosure. The following table shows the reminder frequency until resolution of each severity classification:
+
+========    =====   ==================
+Severity    Score   Reminder frequency
+========    =====   ==================
+Low         ≥0.1    Once a semester
+Medium      ≥4.0    Once a quarter
+High        ≥7.0    Once a month
+Critical    ≥9.0    Once a week
```

Comment:
At first glance this seemed to infrequent but given that after the initial notification it's the maintainer's responsibility to prioritize and resolve the issue, I think it makes sense.